msetnxex.lua

```diff
@@ -0,0 +1,11 @@
+local setExCount = 0
+if redis.call("msetnx", unpack(KEYS)) == 1 then
```

Comment:
This is not right, please see below. you need to construct a new array using `KEYS` with `ARGV[1]` instead

Reply:
Updated !